java/arcs/core/common/BUILD

```diff
@@ -14,3 +15,14 @@ kt_jvm_and_js_library(
         "//java/arcs/core/util",
     ],
 )
+
+arcs_kt_native_library(
+    name = "common-wasm",
+    # Id contains references to Kotlin's Random library, which isn't available
+    # in wasm. So this src list is targeting only the pieces of common that
+    # are needed on wasm as well. It probably makes sense to reorganize
+    # packages at some point.
+    srcs = glob([
+        "Referencable.kt",
+    ]),
```

Comment:
Any objections to doing this, at least for the time being?

Reply:
Looks OK for now